src/resolvers/locations.ts

```diff
@@ -0,0 +1,59 @@
+import { BaseTypeResolver, Languages } from '../types/graphql';
+import { ObjectId } from 'mongodb';
+import { filterEntityFields } from '../utils';
+
+/**
+ * Multilingual location fields
+ */
+const multilingualPersonFields = [
+  'name',
+  'description'
```

Comment:
Не хватает архитекторов
```suggestion
  'description',
  'architects'
```